facter.gemspec

```diff
@@ -8,6 +8,7 @@ Gem::Specification.new do |spec|
   spec.version       = '4.0.51'
   spec.authors       = ['Puppet']
   spec.email         = ['team-nw@puppet.com']
+  spec.homepage      = "https://github.com/puppetlabs/facter"
```

Comment:
Thank you for your contribution!

To pass rubocop checks, could you please use single quotes? Also, could you update your commit message to something like: `(docs) Add homepage metadata to gemspec` ?

Reply:
sure... and done